app/ai_backend/test_genesis_connector.py

```
@@ -1151,30 +2108,21 @@ def test_get_headers_with_additional_headers(self):
             
-            # Start a request in a thread
-            import threading
-            request_thread = threading.Thread(
-                target=self.connector.send_request,
-                args=({'message': 'test'},)
-            )
-            request_thread.start()
-            
-            # Close connection
-            result = self.connector.close()
+            payload = {'message': 'resource_test'}
```

Ruff (0.11.9)

Local variable `payload` is assigned to but never used

Remove assignment to unused variable `payload`

(F841)